test/e2e-tests-upgrade.sh

```diff
@@ -41,17 +42,22 @@ header "Setting up environment"
 set +o errexit
 set +o pipefail
 
+export SYSTEM_NAMESPACE=tekton-pipelines
+
 # First, we will verify if Scenario 1 works.
 # Install the previous release.
 header "Install the previous release of Tekton pipeline $PREVIOUS_PIPELINE_VERSION"
 install_pipeline_crd_version $PREVIOUS_PIPELINE_VERSION
 
+failed=0
+# The upgrade test for the old server version to prevent regression on existing changes.
```

Comment:
```suggestion
# Run upgrade tests for the old server version to prevent regressions where a new client is incompatible with an old server (e.g. https://github.com/tektoncd/pipeline/issues/4913)
```

Reply:
Thanks Lee!